src/builders/interface.jl

```diff
@@ -55,6 +55,10 @@ function interface_constraints(
         prep(id, (mod, mod[interface]))
     prep(id::Symbol, (mod, interface, multiplier)::Tuple{C.ConstraintTree,Symbol,<:Real}) =
         prep(id, (mod, C.map(c -> c * multiplier, mod[interface])))
+    prep(
+        id::Symbol,
+        (mod, interface, multiplier)::Tuple{C.ConstraintTree,C.ConstraintTree,<:Real},
+    ) = prep(id, (mod, C.map(c -> c * multiplier, interface)))
```

Comment:
this is fixed differently in my branch, pls rebase

Reply:
reverted and removed from this branched